### PR TITLE
Control service: remove deprecated dependency 

### PR DESCRIPTION
**Pull request by murphp15**
# Why
At the moment we are blocked on spring upgrades because we are using a number of legacy classes. https://github.com/vmware/versatile-data-kit/pull/1530
This remove one of those dependencies. 
Even after this is merged I don't expect that PR to be fixed as there will be a number of other issues but this is a good first step
# What
I followed the guidelines here on how to remove it https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter

Closes: https://github.com/vmware/versatile-data-kit/issues/1590

Signed-off-by: murphp15 <murphp15@tcd.ie>

### CODE REVIEW

**Comment by DeltaMichael** [comment]
This seems to follow the migration guide closely, so no problem with the code itself. I'm just curious if and how the security config is tested.

**Review by ivakoleva — APPROVED** [review]
mind that merging may then auto-close the issue;

**Review by mivanov1988 — APPROVED** [review]

**Comment by mivanov1988** [comment]
LGTM

File: projects/control-service/projects/base/src/main/java/com/vmware/taurus/security/SecurityConfiguration.java [hunk 1]
```diff
@@ -111,22 +112,20 @@ public SecurityConfiguration(
...
-      enableSecurity(http);
-    } else {
-      log.info("Security is disabled.");
-      http.csrf().disable().authorizeRequests().anyRequest().anonymous();
-    }
+  @Bean
+  @Primary
+  protected SecurityFilterChain configure(
+      HttpSecurity http, AuthenticationManager authenticationManager) throws Exception {
+    return enableSecurity(http, authenticationManager);
```
Comment (murphp15) [inline projects/control-service/projects/base/src/main/java/com/vmware/taurus/security/SecurityConfiguration.java:119]: I'm looking into why I removed the else statement here before merging.

**Pull request merged**